maintainers/maintainer-list.nix

```diff
@@ -7249,6 +7255,12 @@
     githubId = 40049608;
     name = "Andy Chun";
   };
+  noreferences = {
+    email = "norkus@norkus.net";
+    github = "noreferences";
+    githubId = 13085275;
+    name = "Juozas Norkus";
+  };
```

Comment:
cc @noreferences, do agree to be added?

Reply:
I don't know if he's seen this, but he did consent in the previous PR: https://github.com/NixOS/nixpkgs/pull/121061#discussion_r622498157